client/src/components/sidebar/friends/RemoveFriend.tsx

```diff
@@ -0,0 +1,146 @@
+import { Alert, Button, Dialog, Popover, Snackbar, Typography } from '@mui/material';
+import { styled } from '@mui/system';
+import { useState } from 'react';
+
+export const emptyProfile = 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png';
+
+const RemoveFriendDialog = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialog-paper': {
+    padding: theme.spacing(1),
+    borderRadius: theme.shape.borderRadius,
+    minWidth: '400px',
+    minHeight: '250px',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+}));
```

Comment:
the modal ui looks a bit off here's a fix:

```suggestion
const RemoveFriendDialog = styled(Dialog)<{ isMobile: boolean }>(({ theme, isMobile }) => ({
  '& .MuiDialog-paper': {
    borderRadius: theme.shape.borderRadius,
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    maxWidth: isMobile ? '100%' : '30%',
  },
}));
```

this fix will allow the modal to be more similar to the figma mock (with other changes seen below as suggestions)